install_config/deploying_cfme.adoc

```diff
@@ -0,0 +1,510 @@
+[[install-config-deploying-cfme]]
+= Deploying Red Hat CloudForms on {product-title}
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+[NOTE]
+====
+Deploying Red Hat CloudForms on {product-title} is a Technology Preview feature
+only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not
+supported with Red Hat production service level agreements (SLAs), might not be
+functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+The {product-title} installer includes an Ansible role and playbook for
+deploying Red Hat CloudForms 4.5 (CloudForms Management Engine 5.8, or CFME) on
+{product-title}. This deployment must be run on a dedicated {product-title}
+cluster, meaning that no applications or infrastructure other than those
+required for the CFME deployment should be running on it.
+
+This role is based on the work in the upstream
+link:https://github.com/ManageIQ/manageiq-pods[*manageiq/manageiq-pods*]
+project. For additional details on configurations specific to ManageIQ (MIQ),
+such as optional post-installation tasks, see the project's
+link:http://manageiq.org/docs/get-started/basic-configuration[upstream documentation].
+
+[NOTE]
+====
+The terms CFME and MIQ are interchangeable in this deployment.
+====
+
+[[deploying-cfme-requirements]]
+== Requirements
+
+[[deploying-cfme-prerequisites]]
+=== Prerequisites
+
+- A running {product-title} 3.6 cluster
+- NFS or other compatible persistent volume storage provider
+- A *cluster-admin* user (created by this role, if required)
+
+[[deploying-cfme-cluster-sizing]]
+=== Cluster Sizing
+
+In order to avoid random deployment failures due to insufficient resources, the
+following minimum cluster size is recommended for a test environment:
```

Comment:
Is "for a test environment" correct? Should be something like "for a production environment"?